openlibrary/core/cache.py

```diff
@@ -554,7 +555,7 @@ def json_encode(self, value):
 
         memcache doesn't like spaces in the key.
         """
-        return json.dumps(value, separators=(",", ":"), sort_keys=True)
+        return json.dumps([] if isinstance(value, Nothing) else value, separators=(",", ":"), sort_keys=True)
```

Comment:
Let's revert this change.  This looks like it's only used to help generate cache keys, and is out of scope.